include/rocksdb/sst_file_reader.h

```diff
@@ -30,6 +31,9 @@ class SstFileReader {
   // If "snapshot" is nullptr, the iterator returns only the latest keys.
   Iterator* NewIterator(const ReadOptions& options);
 
+  FragmentedRangeTombstoneIterator* NewRangeTombstoneIterator(
```

Comment:
Is this specific to user-defined timestamp?

Reply:
It's not, and it is only used in unit test. It seems previously we did not test sst_file_reader regarding range deletion.